### DISCUSSION TITLE
Express LRS and Frsky R9M Tx modue 900 mhz

### DISCUSSION THREAD

**Discussion by FrankDrebin88** [ExpressLRS/ExpressLRS#2043] [General]
Very good evening, i m wondering if anyone have already flashed the frsky r9m 900 mhz long range module with express lrs firmware and if the module can be used at full power or it need a cooling fan.

Kind Regards 

Daniele

**Comment by greenbigfrog** [comment]
Yes. It's even mentioned in the documentation: https://www.expresslrs.org/3.0/quick-start/transmitters/frsky-r9modules/
And no, you can't use it at full power (for longer duration) without active cooling. https://www.expresslrs.org/3.0/hardware/fan-mod/

**Comment by arfk9919** [comment]
I want to ask if there is an alternative to the transistor that I marked is damaged
![20230303_235642](https://user-images.githubusercontent.com/126906193/222798722-0109797d-d7e4-4dbf-a7d4-939b9abae08f.jpg)

**Reply by avio-byte** [reply]
I have damaged the same Transistor, i was wondering if you found a solution to that

i was forced to make a jumper cable for the module to run from the JR Bay's Voltage, and flash ELRS software using ST-LINK but my module wont run on the Lua script of the radio.
